backend/endpoints/rooms.py

```diff
@@ -62,3 +63,75 @@ async def create_room(
         status_code=status.HTTP_424_FAILED_DEPENDENCY,
         detail="unable to create room",
     )
+
+
+@router.put(
+    "/org/{org_id}/rooms/{room_id}/members/{member_id}",
+    status_code=status.HTTP_200_OK,
+    responses={
+        400: {"detail": "the max number for a Group_DM is 9"},
+        401: {"detail": "member not an admin"},
+        403: {"detail": "DM room or not found"},
+    },
+)
+async def add_to_room(
+    data: AddToRoom,
+    org_id: str,
+    room_id: str,
+    member_id: str,
+    background_tasks: BackgroundTasks,
+):
+    """Adds a new member(s) to a room
+    Args:
+        data: a pydantic schema that defines the request params
+        org_id (str): A unique identifier of an organisation
+        room_id: A unique identifier of the room to be updated
+        member_id: A unique identifier of the member initiating the request
+
+    Returns:
+        HTTP_200_OK: member added
+    Raises:
+        HTTP_400_BAD_REQUEST: the max number for a Group_DM is 9
+        HTTP_401_UNAUTHORIZED: member not in room or not an admin
+        HTTP_403_FORBIDDEN: DM room or not found
+    """
+
+    DB = DataStorage(org_id)
+    new_member = data.dict().get("new_member")
+    room = await get_room(org_id=org_id, room_id=room_id)
+    member = room.get("room_members").get(str(member_id))
+
+    if room is None or room["room_type"] == RoomType.DM:
+        raise HTTPException(status_code=403, detail="DM room or not found")
+
+    if member is None or member["role"] != Role.ADMIN:
+        raise HTTPException(
+            status_code=401, detail="member not in room or not an admin"
+        )
+
+    if room["room_type"] == RoomType.CHANNEL:
+        room["room_members"].update(new_member)
+
+    if room["room_type"] == RoomType.GROUP_DM:
+        room["room_members"].update(new_member)
+        if len(room["room_members"].keys()) > 9:
+            raise HTTPException(
+                detail="the max number for a Group_DM is 9",
+                status_code=400,
+            )
+
+    update_members = {"room_members": room["room_members"]}
+    update_res = await DB.update(
```

Comment:
Here i suggest we write 
```suggestion
    update_response = await DB.update(
```
OR
```suggestion
    response = await DB.update(
```